package.json

```diff
@@ -16,6 +16,7 @@
   "license": "ISC",
   "peerDependencies": {
     "babel-cli": "6.26.0",
+    "babel-plugin-styled-components": "1.4.0",
```

Comment:
Why is this a peer dependency? I thought we compiled before any includes elsewhere.

Reply:
Because you told me to add it as a peer dependency :D

Reply:
I might have been wrong. Or maybe I'm wrong now. Which one is it? :p

Reply:
We use this in wolverine anyway, so remove it?